include/ninshiki_cpp/node/ninshiki_cpp_node.hpp

```diff
@@ -81,7 +76,7 @@ class NinshikiCppNode
   ConfigGrpc config_grpc;
   std::string path;
 
-  static std::string get_node_prefix();
+  static const std::string get_node_prefix();
```

Comment:
No need to specify `const` in here. Actualy, it would be better to pass it as a `std::stringview`. But nevermind, we shouldn't use this function, specifying the node prefix directly is better.